examples/distributed_stencil/CMakeLists.txt

```diff
@@ -0,0 +1,32 @@
+# Copyright (c) 2020 Nikunj Gupta
+#
+# SPDX-License-Identifier: BSL-1.0
+# Distributed under the Boost Software License, Version 1.0. (See accompanying
+# file LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt)
+
+if(NOT HPX_WITH_DISTRIBUTED_RUNTIME)
+  return()
+endif()
+
+set(example_programs 1d_stencil)
```

Comment:
I think we have to rename this. How about `1d_stencil_channel` and putting it in the `examples/1d_stencil` directory? There are distributed 1d stencil implementations in that directory as well, and it might be useful to make it clear what this one does compared to those. Having them together would make that easier.